src/lib/projects/YearlyEvents.svelte

```diff
@@ -19,18 +47,26 @@
 			<label class="ml-4" for="only-members">Nur Mitglieder</label>
 		</div>
 	</div>
-	<section class="mt-12">
-		<img src={EventImage} alt="" class="mt-12 rounded-t-2xl" />
-		<div class="p-6 rounded-b-2xl bg-soft-yellow flex">
-			<div class="flex flex-col justify-center items-center">
-				<h1 class="text-5xl font-serif font-bold">24</h1>
-				<h1>August</h1>
+	{#each events as event}
+		<section class="mt-12">
+			<img src={event.image} alt="" class="mt-12 rounded-t-2xl" />
+			<div
+				class="p-6 rounded-b-2xl flex shadow-lg {event.for_all
+					? 'bg-soft-yellow'
+					: 'bg-light-green'}"
+			>
+				<div class="mt-4 flex flex-col items-center">
+					<h1 class="text-5xl font-serif font-bold">{event.day}</h1>
+					<h1>{event.month}</h1>
+				</div>
+				<div class="ml-10">
+					<h1 class="font-bold">{event.title}</h1>
```

Comment:
heading have a hierarch of `h1` > `h2` > `h3` > ... > `h6`

So this should probably be `h2`.

Note: I just read [this note](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page) by mozilla saying that there should only be a single `h1` on a page. I've not done it that way on the other pages but would like us to adopt this best-practice.

I'll look into automating that with a lint/style check perhaps.

Reply:
I changed the event title to `h2`.
So in the YearlyEvents.svelte is only  a single `h1`, but in the page is another `h1` from the Hero.svelte.